virttest/utils_test/__init__.py

```diff
@@ -1755,46 +1755,48 @@ def __str__(self):
 class VMStress(object):
 
     """
-    Run Stress tool in vms, such as stress, unixbench, iozone and etc.
+    Run Stress tool in vms, such as stress, iozone and etc.
     """
 
-    def __init__(self, vm, stress_type):
-        """
-        Set parameters for stress type
+    def __init__(self, vm, stress_type, download_url="", make_cmds="",
+                 uninstall_cmds="", work_path="", stress_cmds="",
```

Comment:
for iozone, no "uninstall_cmds" needed. Suggest:
  def __init__(self, vm, stress_type, download_url="", make_cmds="", work_path="", stress_cmds="", stress_args="",  uninstall_cmds="", download_type="tarball") 
or something like that.

Reply:
@aliang123 Get your point, move it backward. And also move work_path backward, since stress does not need it. Thanks.